sysfs/class_fibrechannel.go

```diff
@@ -153,6 +153,10 @@ func parseFibreChannelStatistics(hostPath string) (*FibreChannelCounters, error)
 	var counters FibreChannelCounters
 
 	path := filepath.Join(hostPath, "statistics")
+	if _, err := os.Stat(path); os.IsNotExist(err) {
+		fmt.Printf("Directory %s does not exist, skipping.\n", path)
```

Comment:
Please do not put `Printf` statements in this library.

Reply:
I have removed the corresponding Printf statements. Please review again. Thank you!